// Copyright 2020 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

option optimize_for = LITE_RUNTIME;

package reporting.test;

// A simple test proto with a string in it.
message TestMessage {
  optional string test = 1;
}
